docs/book/src/tasks/experimental-features/experimental-features.md

```diff
@@ -0,0 +1,57 @@
+# Experimental Features
+
+Cluster API now ships with a new experimental package that lives under exp/ directory which has new features. This is a
+temporary location for features which will be moved to their permanent locations after graduation. Users can experiment with these features by enabling them using feature gates.
+
+## Enabling Experimental Features for Management Clusters Started with clusterctl
+
+Users can enable/disable features using OS environment variables (`export EXP_CLUSTER_RESOURSE_SET=true`), but it is also possible to set them in the clusterctl config file located at`$HOME/.cluster-api/clusterctl.yaml`:
```

Comment:
I think we should break down this sentence to make it more clear

Users can enable/disable features by setting OS environment variables **before** running `clusterctl init`, e.g:
```
export EXP_CLUSTER_RESOURSE_SET=true

clusterctl init --infrastructure vsphere
```
As an alternative to environment variables, t is also possible to set variables in the clusterctl config file located at`$HOME/.cluster-api/clusterctl.yaml`, e.g.: